### PR TITLE
Rename to 'Files'

### PR DESCRIPTION
**Pull request by marusak**
This also requires to rename this repo. Please debate :)

### CODE REVIEW

**Review by KKoukiou — APPROVED** [review]

File: test/check-application [hunk 1]
```diff
@@ -571,7 +571,7 @@ class TestNavigator(testlib.MachineCase):
             b.select_from_dropdown("#edit-permissions-group-access", access)
             b.select_from_dropdown("#edit-permissions-other-access", access)
 
-        self.enter_navigator()
+        self.enter_file-manager()
```
Comment (KKoukiou) [inline test/check-application:574]: I think you need underscore here.

**Review by KKoukiou — CHANGES_REQUESTED** [review]

**Comment by martinpitt** [comment]
I'm ok with changing the name. Bots [dislike some syntax thing](https://cockpit-logs.us-east-1.linodeobjects.com/pull-291-20240229-145049-9e09551e-debian-testing/log.html).

What I don't like is the "manager" bit (I know, that's odd , with this PR coming from you @marusak :grin: ). We don't call our pages "accounts manager", "VM manger", or "container manager". So if we rename it, I'd rather go with "cockpit-files". But then it supports directories too :grin: 

This is further complicated by navigator being an existing thing in Fedora and other places, so this would need to grow some `Obsoletes:`, `Replaces:` and such.

So in the end

 - Naming is hard
 - Is it really worth it? Where is this coming from?

**Comment by marusak** [comment]
>What I don't like is the "manager" bit (I know, that's odd , with this PR coming from you @marusak 😁 )

Should I take this personally? :rofl: 

> We don't call our pages "accounts manager", "VM manger", or "container manager". So if we rename it, I'd rather go with "cockpit-files". But then it supports directories too 😁

Good point!

> This is further complicated by navigator being an existing thing in Fedora and other places, so this would need to grow some Obsoletes:, Replaces: and such.

I want to introduce that in separate PR. Actually, this makes it easier as if we want to start releasing into existing packages we need some rights to do that.

> Is it really worth it? Where is this coming from?

There was already question from someone what is "navigator". It is not clear from this repo (go to https://github.com/cockpit-project/cockpit-navigator and figure out what it does :)) but also we call it 'File browser' in the menu so the link between name and package name already is lost.

> Naming is hard

'File browser'? :) 
'Files'?

**Comment by allisonkarlitskaya** [comment]
2¢: The name of the article for (this concept) on Wikipedia is "File manager" — I don't know any gut-feeling source more authoritative than that.  Files is a fine name.  Otherwise, looking at the names of various software that does this sort of thing (ie: appeal to user familiarity), one can find a lot of repeated uses of the words "commander" (particularly for the old-school dual-pane ones), "explorer", and "tree".  Of those three, I think "cockpit-explorer" is the only one that vaguely fits, even if "cockpit-commander" sounds kinda l33t.

Explorer has the benefit of being familiar to nearly all Windows users as well as many Android users (as nearly all of the most popular file manager apps there have names like "Mixplorer", "Solid Explorer", "X-plore", "FX Explorer", etc.) ((source: the reddit post which was the first google hit))

**Comment by jelly** [comment]
Cockpit-Files also SGTM

**Comment by garrett** [comment]
I support renaming.

It's a general design rule of thumb to really avoid "manager" in interfaces.

I'm fine with "Cockpit-Files" as the project name and "Files" in the sidebar. It makes sense, especially since we don't use "Accounts manager" / "Accounts browser" or any other "manager" or "browser" elsewhere — and nor should we. 

(Of course, the "browser" itself is really Cockpit Client, Firefox, Chrome, Safari, etc., right? :wink:)

**Comment by marusak** [comment]
Ok, renamed to Files. @martinpitt @allisonkarlitskaya @KKoukiou @garrett We all agree? :) 

It has fresh conflict, I will resolve but we need to land quickly then as it is easy to get conflicts

**Comment by martinpitt** [comment]
I'm fine with "Cockpit Files", thanks!

**Comment by jelly** [comment]
Same, we do need to update the localStorage settings after https://github.com/cockpit-project/cockpit-navigator/pull/3226

File: packaging/debian/control [hunk 2]
```diff
@@ -7,11 +7,11 @@ Build-Depends: debhelper-compat (= 13),
...
 Rules-Requires-Root: no
-Homepage: https://github.com/cockpit-project/cockpit-navigator
-Vcs-Git: https://salsa.debian.org/debian/cockpit-navigator.git
-Vcs-Browser: https://salsa.debian.org/debian/cockpit-navigator
+Homepage: https://github.com/cockpit-project/cockpit-files
+Vcs-Git: https://salsa.debian.org/debian/cockpit-files.git
+Vcs-Browser: https://salsa.debian.org/debian/cockpit-files
 
-Package: cockpit-navigator
+Package: cockpit-files
```
Comment (jelly) [inline packaging/debian/control:14]: No replaces?

File: packaging/cockpit-files.spec.in [hunk 3]
```diff
@@ -1,11 +1,11 @@
-Name: cockpit-navigator
+Name: cockpit-files
```
Comment (jelly) [inline packaging/cockpit-files.spec.in:1]: This should replace navigator no?

**Review by jelly — CHANGES_REQUESTED** [review]

**Comment by jelly** [comment]
This is a new flake https://cockpit-logs.us-east-1.linodeobjects.com/pull-291-7e9d6b0f-20240315-093324-fedora-39-devel/log.html#10

**Comment by marusak** [comment]
@jelly retried. I have possible fix I will push in separate PR.

**Review by allisonkarlitskaya — APPROVED** [review]

**Review by martinpitt — DISMISSED** [review]

**Review by KKoukiou — DISMISSED** [review]

File: src/files-folder-view.tsx [hunk 4]
```diff
@@ -36,31 +36,31 @@ export const NavigatorFolderView = ({
     addAlert,
 }: {
     path: string[],
-    files: NavigatorFileInfo[],
+    files: FilesFileInfo[],
     loadingFiles: boolean,
     showHidden: boolean,
-    selected: NavigatorFileInfo[],
-    setSelected: React.Dispatch<React.SetStateAction<NavigatorFileInfo[]>>,
+    selected: FilesFileInfo[],
```
Comment (jelly) [inline src/files-folder-view.tsx:42]: This makes no sense :)

File: src/manifest.json [hunk 5]
```diff
@@ -8,7 +8,7 @@
           "label": "File browser",
           "keywords": [
 		{
-			"matches": ["files", "navigator", "explorer", "filesystem"]
+			"matches": ["files", "files", "explorer", "filesystem"]
```
Comment (jelly) [inline src/manifest.json:11]: files twice now

**Review by jelly — APPROVED** [review]

**Pull request merged**